.quickstart/quickstart.yml

```diff
@@ -1,4 +1,4 @@
-dbt_versions: '>=1.3.0 <2.0.0'
+dbt_versions: '>=1.6.0 <2.0.0'
```

Comment:
Let's also update the badge in the README to show the proper version range.
<img width="277" height="53" alt="image" src="https://github.com/user-attachments/assets/8507980c-8856-43a4-b7ad-8e4f94a04985" />

Reply:
Updated